must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -250,12 +250,25 @@ for ns in $namespaces; do
         blockpools=$(timeout 60 oc get cephblockpools.ceph.rook.io -n openshift-storage -o jsonpath="{range .items[*]}{@.metadata.name}{'\n'}{end}")
         for bp in $blockpools; do
             images=$(timeout 60 oc -n openshift-storage exec "${HOSTNAME}"-helper -- bash -c "rbd ls -p $bp")
+            pids_rbd=()
             for image in $images; do
-                { timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd info $image --pool $bp" >> "${COMMAND_OUTPUT_FILE}"; } >> "${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-image-"${image}"-debug.log 2>&1 &
-                pids_ceph+=($!)
-                { timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd snap ls --all $image --pool $bp" >> "${COMMAND_OUTPUT_FILE}"; } >> "${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-snap-"${image}"-debug.log 2>&1 &
-                pids_ceph+=($!)
+                printf "collecting vol and snapshot info for %s\n" "${image}" | tee -a  "${BASE_COLLECTION_PATH}"/gather-ceph-debug.log
+                COMMAND_OUTPUT_FILE_VOL=${COMMAND_OUTPUT_DIR}/rbd_vol_and_snap_info_${image}.part
+                { 
+                    printf "Collecting image info for: %s\n" "${image}";
+                    timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd info $image --pool $bp" 2>> "${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-image-"${image}"-debug.log;
+                    printf "Collecting snap info for: %s\n" "${image}";
+                    timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd snap ls --all $image --pool $bp" 2>> "${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-snap-"${image}"-debug.log; 
+                } >> "${COMMAND_OUTPUT_FILE_VOL}" &
+                pids_rbd+=($!)
             done
+            if [ -n "${pids_rbd[*]}" ]; then
+                # wait for all pids
+                echo "waiting for ${pids_rbd[*]} to terminate" | tee -a  "${BASE_COLLECTION_PATH}"/gather-debug.log
+                wait "${pids_rbd[@]}"
+            fi
+            cat "${COMMAND_OUTPUT_DIR}"/rbd_vol_and_snap_info_*.part >> "${COMMAND_OUTPUT_FILE}"
+            rm -f "${COMMAND_OUTPUT_DIR}"/rbd_vol_and_snap_info_*.part
```

Comment:
It might be better to do this in a loop to avoid a really long command line for clusters with a lot of images (as each name is going to be ~75 chars + `COMMAND_OUTPUT_DIR` prefix long).